test/test_issues/test_i219/expect/mod1.expect

```diff
@@ -0,0 +1,3 @@
+mod1.yang:9: error: INVALID_CONFIG
```

Comment:
Perhaps this is not due to your code, but this error is a bit weird - it refers to line 9, but line 9 is fine, the problem is on line 38.  Somehow we need to update the .pos with the deviation statement's .pos.

Reply:
OK, I will replace the .pos with the deviation statement's .pos.